src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -298,7 +294,17 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IPublicationService, PublicationService>();
             services.AddTransient<IPublicationRepository, PublicationRepository>();
             services.AddTransient<IMetaService, MetaService>();
-            services.AddTransient<ILegacyReleaseService, LegacyReleaseService>();
+            services.AddTransient<ILegacyReleaseService, LegacyReleaseService>(provider =>
```

Comment:
Same problem exists with `_publicBlobCacheService` that was recently added to `PublicationService`.

Also because of this, while we can easily clear the cached tree json blobs, I don't think we have a way of clearing all publication json blobs that may be now out of date without using the publisher functions method of republishing all content.